ios/RNAudioRecorderPlayer.swift

```diff
@@ -224,31 +271,8 @@ class RNAudioRecorderPlayer: RCTEventEmitter, AVAudioRecorderDelegate {
             }
         }
 
-
-        if (numberOfChannel == nil) {
-            numberOfChannel = 2
-        }
-
-        if (audioQuality == nil) {
-            audioQuality = AVAudioQuality.medium.rawValue
-        }
-
-        if (bitRate == nil) {
-            bitRate = 128000
-        }
-
         func startRecording() {
-            let settings = [
-                AVSampleRateKey: sampleRate!,
-                AVFormatIDKey: avFormat!,
-                AVNumberOfChannelsKey: numberOfChannel!,
-                AVEncoderAudioQualityKey: audioQuality!,
-                AVLinearPCMBitDepthKey: avLPCMBitDepth ?? AVLinearPCMBitDepthKey.count,
-                AVLinearPCMIsBigEndianKey: avLPCMIsBigEndian ?? true,
-                AVLinearPCMIsFloatKey: avLPCMIsFloatKey ?? false,
-                AVLinearPCMIsNonInterleaved: avLPCMIsNonInterleaved ?? false,
-                AVEncoderBitRateKey: bitRate!
-            ] as [String : Any]
+            let settings = customSettings.count > 0 ? customSettings : defaultSettings
```

Comment:
Wouldn't this ignore the `default` values when one of `customSettings` provided? For example, when `AVSampleRateKey` is provided, what are other default values be?